packages/flow-remove-types/test/source.js

```diff
@@ -1,6 +1,8 @@
 /* @flow */
 // @nolint
 
+// multi-byte chars: Гарного дня, котики!
```

Comment:
Can we add some non-BMP (multi UTF-16 code unit) characters to the test? e.g. emoji: `'🐈'.length == 2`, `Buffer.from('🐈').length == 4`

Also, I'd add a test case with non-ASCII characters in the actual code that we process, e.g.
```js
var lambda: λ = (α: number): number => α;
```